### PR TITLE
Update report-view.md

### PR DESCRIPTION
**Pull request by dirk-pieterse**

### CODE REVIEW

File: documentation/user-interface/report-view.md [hunk 1]
```diff
@@ -35,13 +35,13 @@ TestCafe Studio provides the following report formats out of the box:
...
-See [Get More Report Formats](#get-more-report-formats) to learn how to make more report formats available.
+See [Get More Report Formats](#get-more-report-formats) for information on how to make add more report formats.
 
-You can select the default format in [run configuration](run-configuration-dialog.md) settings.
+You can select the default format in the [run configuration](run-configuration-dialog.md) settings.
 
 ### Work with a Grid
 
-Grid view is the most powerful report format to study and analyze.
+The Grid view is the most powerful report format to study and analyze.
```
Comment (dirk-pieterse) [inline documentation/user-interface/report-view.md]: "most powerful report format to study and analyze" This is incomplete and does not really tell the reader anything useful about the Grid View.

File: documentation/user-interface/report-view.md [hunk 2]
```diff
@@ -22,11 +22,11 @@ The **Report** view displays a detailed report about a test run. It allows you t
 
 ## Analyze Test Results
 
-The **Report** view can present data in different formats. They include machine-readable formats like JSON or formats that better fit for human examination like a grid view. Use a drop-down list in the report header to select a format.
+The **Report** view can display data in different formats. You can use the report header's drop-down list to select a format.
```
Comment (dirk-pieterse) [inline documentation/user-interface/report-view.md]: The whole Analyze Test Results section is too wordy. You can combine and reorder the information in this section to improve the readability and reduce the number of words. The part about 'better fit for human examination' is strange, and the image already shows the available formats.

Comment (VasilyStrelyaev) [reply documentation/user-interface/report-view.md]: > The whole Analyze Test Results section is too wordy. You can combine and reorder the information in this section to improve the readability and reduce the number of words. The part about 'better fit for human examination' is strange...

okay, I see

> the image already shows the available formats.

well, I don't treat images as a reliable source of information. They should be used only for illustrative purposes. In this case, showing where the drop-down can be invoked and how it looks like, While important information like the list of available report formats should be written in the text (even if it duplicates the screenshots).

File: documentation/user-interface/report-view.md [hunk 3]
```diff
@@ -68,7 +68,7 @@ Click the ![Screenshot Icon](../../images/user-interface/report-view/screenshot-
 
 #### Examine Failed Tests
 
-If a test has failed, an arrow is displayed near its status. This arrow allows you to expand the grid row and see details about this fail.
+An arrow is displayed near a test's status to indicate if it failed. This arrow allows you to expand the grid row and see details about this failure.
```
Comment (VasilyStrelyaev) [inline documentation/user-interface/report-view.md]: An arrow does not actually indicate that a test failed. We have an icon for that. The arrow just allows you to expand the row and see the detailed info.
